Terms of Reference.md

```diff
@@ -0,0 +1,140 @@
+- [Membership Composition](#membership-composition)
+- [Project Schedule](#project-schedule)
+- [Meeting Schedule](#meeting-schedule)
+- [Consensus Building](#consensus-building)
+- [Code of Conduct](#code-of-conduct)
+
+## Overview
+
+This Terms of Reference (ToR) defines the goals of the Trust Meter Technical
+Specification, how it differs or relates to other specifications regarding
+Artificial Intelligence (AI) and Trust, and the objectives of the Technical
+Committee, outlining the roles and responsibilities of those involved.
+
+## Project Scope
+
+The Trust Meter Technical Specification addresses statistical discrimination
+against data outliers and small minorities in mechanized statistical reasoning
+in AI decision tools. The Trust Meter Technical Specification is intended to
+alert AI operators when the scenario, group or individual about whom the
+decision is made is out-of-distribution relative to the training set the model
+is trained on, meaning the AI should not be trusted to make decisions in this
+context.
+
+## Purpose of the Committee
+
+The overall objective of the Technical Committee is to determine how to evaluate
+an AI tool's decision making algorithm when used with marginalized or outlier
+individuals; to detect whether the use of the AI discriminates against them.
+This main objective leads to some sub-goals.
+
+The committee will develop definitions of "marginalized" and "outlier" to fully
+contextualize their importance and relevance to the evaluation of an AI model.
+This will help operationalize the notion of statistical discrimination. 
+Considerations in this regard include the reasons why an individual is deemed
+marginalized or an outlier.  A possible reason, or example, is that they were
+either not part of the training data or were poorly represented within it.
+Another reason might be the nature of the model or algorithm that the AI uses in
+making its decisions.  That model or algorithm might produce discriminatory
+results in the context of a marginalized individual and that would indicate it
+is inappropriate to use in that case.
+
+The committee will also delineate the kinds of generative AI that are relevant
+to the specification.  For example, an AI that is definitely out-of-scope is one
+that generates photorealistic images of human faces. This kind of AI is not
+properly described as making any decisions. However, a large language model
+(LLM) may or may be in scope. For example, consider this scenario: An AI
+decision tool is known to discriminate and it uses an LLM during its processing
+to arrive at a decision.  The LLM might be the source of the discrimination;
+hence, the LLM is in scope in this scenario.
+
+The committee will develop and document techniques for evaluating an AI in order
+to determine whether it discriminates or can potentially discriminate.  This
+might involve the methods used in the field of explainable AI. The  techniques
```

Comment:
Remove extra spaces between the last two words in this line: "The" and "techniques".

Reply:
Done -- thanks!